app/templates/download.html

```diff
@@ -0,0 +1,27 @@
+{% from 'macros/page_heading.html' import page_heading %}
```

Comment:
Why do we need a separate page for this? Could we just link directly to the downloads on when we list the frameworks?